src/Raven.Client/Documents/Commands/SeedIdentityForCommand.cs

```diff
@@ -1,54 +0,0 @@
-﻿using System;
-using System.Net.Http;
-using Raven.Client.Http;
-using Raven.Client.Util;
-using Sparrow.Json;
-
-namespace Raven.Client.Documents.Commands
-{
-    public class SeedIdentityForCommand : RavenCommand<long>, IRaftCommand
```

Comment:
We cannot remove those. Need to stay in the same place as they were.

Reply:
done